helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -577,9 +577,36 @@
             }
           }
         },
+        "podSecurityContext": {
+          "type": "object"
+        },
         "securityContext": {
```

Comment:
This can also be deprecated in favor of `containerSecurityContext` but I think we can defer that until fluentd is removed